pkg/api/admin/api/client.go

```diff
@@ -410,6 +410,92 @@ func (client OpenShiftManagedClustersClient) RestoreResponder(resp *http.Respons
 	return
 }
 
+// RotateSecretsAndWait rotates the keys of an openshift managed cluster and waits
+// for the request to complete before returning.
+func (client OpenShiftManagedClustersClient) RotateSecretsAndWait(ctx context.Context, resourceGroupName, resourceName string, parameters OpenShiftManagedCluster) (result autorest.Response, err error) {
```

Comment:
I don't think you need to pass `parameters`? It should be enough to send the rg and the resource name of the cluster that should have its secrets rotated.

Reply:
thats correct. i'll take parameters out.